packages/firestore/src/remote/connection.ts

```diff
@@ -90,7 +90,7 @@ export interface Connection {
    * request message must include the path. If false, then the request message must NOT
    * include the path.
    */
-  get shouldResourcePathBeIncludedInRequest(): boolean;
+  readonly shouldResourcePathBeIncludedInRequest: boolean;
```

Comment:
Do you need to make a corresponding change in grpc_connection.ts and rest_connection.ts?

Reply:
based on Typescript, no. readonly fields is corresponding to getters. https://bobbyhadz.com/blog/typescript-interface-getter-setter.